src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindow.cs

```diff
@@ -179,6 +182,17 @@ private void CompositeGUI()
                             compositionManager.OcclusionMode = occlusionModes[selectedIndex];
 
                             GUI.enabled = true;
+
+                            if (compositionManager.CaptureDevice == FrameProviderDeviceType.AzureKinect && compositionManager.OcclusionMode == OcclusionSetting.BodyTracking && !IsAzureKinectBodyTrackingSDKInstalledInUnity())
+                            {
+                                var previousColor = GUI.backgroundColor;
+                                GUI.backgroundColor = Color.red;
+                                if (GUILayout.Button(new GUIContent("Install Required Components", "Copies components needed for the Azure Kinect Body Tracking into your Unity installation directory (requires elevation)"), GUILayout.Width(250.0f)))
+                                {
+                                    InstallAzureKinectBodyTrackingComponents();
+                                }
+                                GUI.backgroundColor = previousColor;
```

Comment:
Can this install method have a bool return to tell whether it failed or succeeded? Seems like if someone did not accept the popup windows that this would set the button back to default background instead of red.